submissions/Yasya23/holi-style-popup/style.css

```diff
@@ -0,0 +1,163 @@
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+:focus {
+  outline: none;
+}
+
+.navigation {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0.3rem 0.6rem 0.6rem 0.3rem;
+}
+
+.navigation__list {
+  position: relative;
+  display: -webkit-box;
+  display: -ms-flexbox;
```

Comment:
Also please share where did you find it)

Reply:
This is also the Live Sass Compiler in VSCode)

Reply:
so, why did you still not remove it?))

Reply:
Already removed)